CREDITS.txt

```diff
@@ -0,0 +1,65 @@
+Minecraft Forge: Credits/Thank You
```

Comment:
This file along with `LICENSE.txt`, `README.txt` and `changelog.txt` can be removed since they are reminiscentes from the forge mdk and not from our project.